.github/autobuild/ios.sh

```diff
@@ -19,7 +19,17 @@ setup() {
     else
         echo "Installing Qt"
         python3 -m pip install "aqtinstall==${AQTINSTALL_VERSION}"
-        python3 -m aqt install-qt --outputdir "${QT_DIR}" mac ios "${QT_VERSION}" --archives qtbase qttools qttranslations qtmacextras
+        # Install actual ios Qt:
+        python3 -m aqt install-qt --outputdir "${QT_DIR}" mac ios "${QT_VERSION}" --archives qtbase qttools qttranslations
+        if [[ ! "${QT_VERSION}" =~ 5\..* ]]; then
+            # Starting with Qt6, ios' qtbase install does no longer include a real qmake binary.
+            # Instead, it is a script which invokes the mac desktop qmake.
+            # As of aqtinstall 2.0.6 / 03/2022, desktop qtbase has to be installed manually:
+            python3 -m aqt install-qt --outputdir "${QT_DIR}" mac desktop "${QT_VERSION}" --archives qtbase
+            # Additionally, Qt6 has renamed the Mac binary location from clang_64 to macos.
```

Comment:
I know you opened an issue related to this at aqtinstall. Is there anything which needs to be changed here?

Reply:
I have tested the removal in https://github.com/hoffie/jamulus/commit/5adde603213b936539b77ad5b4f82a8f460ff28e and it works. Therefore, I've removed the symlink workaround here as well (the manual desktop install is still required).